### ISSUE TITLE
www.cibc.com/clientlibs

### ISSUE THREAD

**Issue by BearTM** [easylist/easylist#7319]
### List the website(s) you're having issues:

`www.cibc.com - Canadian Banking Website`

### What happens?

```
Blocking /www.cibc.com/clientlibs
Unable to login to Online Banking.
Banking login process is entirely non-functional.
No errors reported. Operation fails with 403 Forbidden request being returned.
```

### List Subscriptions you're using:

```
Block has been recently added to EasyPrivacy
||cibc.com/clientlibs/

Version: 202103021917
Title: EasyPrivacy
Last modified: 02 Mar 2021 19:17 UTC
```

### Your settings

```
- OS/version: Windows 10 (64-bit)
- Browser/version: Chrome Version 88.0.4324.190
- Adblock Extension/version: uBlock Origin v1.33.2
```
### Other details:

```
ryanbr commit - 4 days ago - requires revert
commit b19bf434e3cc5cd896cfd6d64b03590d3fc499ed
```

**Comment by ryanbr** [comment]
https://www.cibc.com/clientlibs/b56a193e8no26195726fae359392f1d  

This is just a fingerprinting script (which then loads additonal checks) Which is loaded separately for the login process.

Same tracking scripts is used on other sites, `att.com`, `discover.com`, `bestbuy.com`, `kohls.com`, `staples.com` just to name a few.

https://github.com/easylist/easylist/blob/master/easyprivacy/easyprivacy_specific.txt#L2701

I can't directly test the site, since I don't have a login. but tested non-banking (above) sites the filters had no affect on the login process.

**Comment by RobBeyer** [comment]
Login was not working for me for a number of days, during which I used the mobile app instead, until I needed the site.

I've disabled the override I added, and checked the login again. Today it's working fine with the block in place.
Yesterday & previously, without the override, it remained on the "spinner" and the operation failed with 403 Forbidden.

They may have had complaints about the login failing and have updated the site.
It appears that the entry is now fine to remain on the list.
